receiver/jmxreceiver/receiver.go

```diff
@@ -155,37 +179,60 @@ func (jmx *jmxMetricReceiver) buildJMXMetricGathererConfig() (string, error) {
 		jmx.config.Endpoint = fmt.Sprintf("service:jmx:rmi:///jndi/rmi://%v:%d/jmxrmi", host, port)
 	}
 
-	javaConfig := fmt.Sprintf(`otel.jmx.service.url = %v
-otel.jmx.interval.milliseconds = %v
-`, jmx.config.Endpoint, jmx.config.CollectionInterval.Milliseconds())
-
-	if jmx.config.TargetSystem != "" {
-		javaConfig += fmt.Sprintf("otel.jmx.target.system = %v\n", jmx.config.TargetSystem)
-	} else if jmx.config.GroovyScript != "" {
-		javaConfig += fmt.Sprintf("otel.jmx.groovy.script = %v\n", jmx.config.GroovyScript)
-	}
+	config["otel.jmx.service.url"] = jmx.config.Endpoint
+	config["otel.jmx.interval.milliseconds"] = strconv.FormatInt(jmx.config.CollectionInterval.Milliseconds(), 10)
+	config["otel.jmx.target.system"] = jmx.config.TargetSystem
```

Comment:
Can we validate the specified value? (split and switch?)

Reply:
Added, in the validate function.